packages/rn-tester/js/examples/LinearGradient/LinearGradientExample.js

```diff
@@ -94,7 +94,7 @@ exports.examples = [
     linear-gradient(45deg, white, rgba(243, 119, 54, 0.8), rgba(243, 119, 54, 0) 70%),
     linear-gradient(90deg, white, rgba(253, 244, 152, 0.8), rgba(253, 244, 152, 0) 70%),
     linear-gradient(135deg, white, rgba(123, 192, 67, 0.8), rgba(123, 192, 67, 0) 70%),
-    linear-gradient(180deg, white, rgba(3, 146, 207, 0.8), rgba(3, 146, 207, 0) 70%);
```

Comment:
the native parser does not support `;` at the end of syntax so i had to change it. I guess it makes sense, we don't need to support semi-colons at the end.